add documentation about how to send alerts from Grafana be e-mail

alimand opened the pull request:
Add documentation about how to send alerts from Grafana be E-mail in Monitoring part:
<img width="480" alt="d4f9dd39-b85b-4ddd-ad26-cb7952ecf483" src="https://github.com/user-attachments/assets/e64016c5-a92f-48ae-ba58-d480abc21dad">
![43e8a016-e8bc-4c37-867c-c4fa0a06497a](https://github.com/user-attachments/assets/cca519a4-253c-47ef-a942-25622624889d)
![ffad2bcf-ad5b-4f99-bf31-de11fe655390](https://github.com/user-attachments/assets/921ca0ad-953d-45ab-add0-bb3024e06f2e)
![70be7cb2-d805-424f-951d-9688119ac446](https://github.com/user-attachments/assets/68bbac64-75eb-416e-82e4-1e1cebff2035)
![849494ee-14af-4d7b-9887-f503e21a6df2](https://github.com/user-attachments/assets/f016e19a-9c6f-4122-b190-ab31716a942f)
![404e34d4-febb-4316-87b2-5dfffc7b2c41](https://github.com/user-attachments/assets/e12e477b-6395-4b95-a03c-cad0adaa458a)

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+.. code-block:: bash
+
+    docker cp grafana:/etc/grafana/grafana.ini /your-local-directory
+This command copies the grafana.ini file from the Docker container to your local machine for editing.
```
Add blank line.

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
Rename page to `docs/source/reference/monitoring/alerts.rst` and making `Receiving alerts by email` a section.

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+
+Copy the grafana.ini File from Docker Container
+-----------------------------------------------
+First, you need to extract the grafana.ini configuration file from your Grafana Docker container to your local system. For example:
```
enclose grafana.ini in a code fence (i.e. `grafana.ini`)

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+Modify the grafana.ini File
+---------------------------
+
+Open the grafana.ini file you just copied in a text editor and locate the [smtp] section. You will need to enable SMTP and configure it to use your email provider's SMTP server. Here’s how you can configure it for an email account:
```
enclose grafana.ini in a code fence (i.e. `grafana.ini`)

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+
+.. note::
+
+   The password used in the grafana.ini SMTP configuration is not your regular email account password. 
```
enclose grafana.ini in a code fence (i.e. `grafana.ini`)

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+   The password used in the grafana.ini SMTP configuration is not your regular email account password. 
+   Detailed descriptions are provided at the bottom of this page.
+
+Mount the Modified grafana.ini File to the Grafana Container
```
rename to "Mount grafana.ini to the Grafana container"

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+   :alt: mount grafana
+   :align: center
+   
+This line tells Docker to use the local version of grafana.ini when starting the Grafana container.
```
enclose grafana.ini in a code fence (i.e. `grafana.ini`)

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+   
+This line tells Docker to use the local version of grafana.ini when starting the Grafana container.
+
+Restart wis2box to Apply Changes
```
Rename to "Applying changes"

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+
+This command stops and then restarts your containers, ensuring that the new configuration is loaded.
+
+Set Up the Notification Channel in Grafana
```
Rename to "Setting up the notification channel in Grafana"

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+
+(1) Navigate to Alerting -> Notification channels.
+
+(2) Click Add channel and choose Email as the notification type.
```
-> Click "Add channel"

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+   :alt: notification channel detail
+   :align: center
+
+Configure Alerts in Your Grafana Dashboard
```
-> Configuring alerts in Your Grafana dashboard

tomkralidis commented on docs/source/reference/monitoring/receive-alerts-by-email.rst:
```diff
@@ -0,0 +1,116 @@
+   :alt: add alert notification
+   :align: center
+
+Final Step: Test the Setup
```
-> Testing the setup

tomkralidis reviewed (changes requested):
Great work @alimand !  See minor change requests.  In addition, please ensure all PNG files in this PR are renamed to not have spaces.

alimand commented:
@tomkralidis Thanks for your time to review Tom, I renamed each png files to make sure there are no spaces anymore and changed the script based on each suggestions.

tomkralidis reviewed (changes requested):
Thanks @alimand.  Some additional comments:

1.
- update all headings/sections without numbers and sentence case do not number headings i.e. changing:

6. Configuring Alerts in Your Grafana Dashboard6. Configuring Alerts in Your Grafana Dashboard

to:

Configuring alerts in your Grafana dashboard

2. Update all numbered lists to bulleted lists i.e. changing:

```
    (1) Enable Less Secure Apps: If you don't use 2FA, enable "Less secure app access" in your Google account settings.
    (2) Generate an App Password: If you use 2FA:
    (3) Go to your Google Account -> Security -> App passwords.
    (4) Generate a new app password for "Mail".
    (5) Use this app password in the grafana.ini password field.
```

to:

```
    * Enable Less Secure Apps: If you don't use 2FA, enable "Less secure app access" in your Google account settings.
    * Generate an App Password: If you use 2FA:
    * Go to your Google Account -> Security -> App passwords.
    * Generate a new app password for "Mail".
    * Use this app password in the grafana.ini password field.
```

3. "Fence" the text "grafana.ini" so that it renders as code:

```
``grafana.ini``
```

alimand commented:
@tomkralidis Thanks Tom, I modified based on additional comments

tomkralidis commented:
Thank you for this contribution @alimand !
